src/components/molecules/Subnavigation/Subnavigation.tsx

```diff
@@ -0,0 +1,36 @@
+import Arrow from "../../../assets/svg/angle-left-solid-black.svg"
+import { MenuData } from "../Menu/Menu";
+import "./Subnavigation.css";
+
+
+
+interface SubnavigationProp {
+    subMenu: MenuData[]
+    backBtnMenu: () => void
+    path?: string;
+    title: string;
+}
+
+const Subnavigation = (props: SubnavigationProp) => {
+    const subnavigationProps = props
+
+    const reduceItem = subnavigationProps.subMenu.reduce<any>((subMenu, item) => {
```

Comment:
Avoid any, let typescript infer subMenu type
Assert the initial value you provide to .reduce to MenuData[]

Reply:
Solved